tests/core/editable/removemultiplelist.js

```diff
@@ -0,0 +1,116 @@
+/* bender-ckeditor-plugins: list,toolbar,table,format */
+
+( function() {
+	'use strict';
+
+	bender.editors = {
+		classic: {},
+		inline: {
+			creator: 'inline'
+		}
+	};
+
+	var tests = {
+		'test remove full multiple list': function( editor, bot ) {
+			bender.tools.testInputOut( 'multiple_full_list', function( source, expected ) {
+				bender.tools.setHtmlWithSelection( editor, source );
+
+				editor.editable().fire( 'keydown', new CKEDITOR.dom.event( { keyCode: 46 } ) );
+
+				bender.assert.beautified.html( expected, bot.htmlWithSelection() );
+			} );
```

Comment:
This block of code is repetitive. It could be extracted to separate testing functions with arguments.